utils/functions

```diff
@@ -1,5 +1,66 @@
 # -*- sh -*-
 
+#M0_SRC_DIR="$(readlink -f $0)"
+#M0_SRC_DIR="${M0_SRC_DIR%/*/*}"
+
+#. $M0_SRC_DIR/scripts/jenkins/build
+
+TRANSPORT=$(m0_default_xprt)
```

Comment:
Move this after definition of m0_default_xprt.

Reply:
Done